### PR TITLE
Document the Pagination component

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

File: docs/src/routes/docs/components/pagination.svx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+name: Pagination
+shadow0: '0 2px 2px 0 rgba(0, 0, 0, .14), 0 1px 5px 0 rgba(0, 0, 0, .12);'
+shadow1: '0 2px 5px rgba(0, 0, 0, .14), 0 1px 5px rgba(0, 0, 0, .12);'
+shadowRaised: '0 2px 4px -1px rgba(0, 0, 0, .2), 0 3px 5px 0 rgba(0, 0, 0, .14), 0 1px 10px 0 rgba(0, 0, 0, .12);'
```
Comment (aabounegm) [inline docs/src/routes/docs/components/pagination.svx:5]: Where are these being used?

**Comment by illright** [comment]
As for the arrows, I don't think this could be fixed easily. We're listening to `change` events

**Comment by aabounegm** [comment]
@illright Couldn't we listen to input events instead and check the pressed key manually?

**Comment by illright** [comment]
I tried listening to the `input` events, it turns out that either HTML or Svelte doesn't fire `input` events for Enter key presses on `type="number"` inputs, so I really don't know how else to implement this. And honestly, it doesn't matter too much to me. We don't show spinners for those inputs anyway, and using the keyboard to navigate random access is strange, imo. So I'm not too worried

**Comment by aabounegm** [comment]
Neither `input` nor `change` get fired for Enter key. The simplest solution would be to listen to `keydown` instead and only react to Enter key presses.

**Comment by aabounegm** [comment]
Also, I'm not sure why you said we are not showing spinners. They are showing for me:
image

**Pull request merged**